sdk/compatibility/src/env.rs

```diff
@@ -54,10 +54,18 @@ impl Environment {
         &self.json_rpc_url
     }
 
+    pub fn websocket_rpc_url(&self) -> String {
+        self.json_rpc_url.replace("http://", "ws://") + "/v1/stream/ws"
```

Comment:
jsonrpc url may end with "/v1", we recommend to use "/v1" url, in case the default root path is upgraded

Reply:
sorry I'm not understanding your comment. The `/v1/` here isn't the same as the jsonrpc specifically- this comes from the websocket implementation, as each transport should manage it's own versioning.